Header header
bool head_up
bool head_down
bool tilt_up
bool tilt_down

